lib/konbata/models/upload_course.rb

```diff
@@ -85,6 +103,12 @@ def upload_content(filename)
       puts "Uploading: #{name}"
       upload_to_s3(migration, filename)
       puts "Done uploading: #{name}"
+
+      if File.exist?(source_for_imscc)
```

Comment:
I don't know if simply checking if the `source_for_imscc` file exists is a great way to make this work. Any suggestions?